webpack.config.js

```diff
@@ -27,5 +28,14 @@ module.exports = {
         ],
       },
     ]
-  }
+  },
+  plugins: [
+    new CompressionPlugin({
```

Comment:
Can you explain what this plugin is doing? Is it compressing your files?

Reply:
Yep! I am using the compression webpack plugin to compress my bundle file. https://www.npmjs.com/package/compression-webpack-plugin